app/rest/model/src/main/resources/syndesis-extension-definition-schema.json

```diff
@@ -0,0 +1,228 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
official json schemas examples (for instance http://json-schema.org/example/geo.json ) contains http://json-schema.org/draft-06/schema#

seems that there is also a 07 version released the 19-11-2017 http://json-schema.org/specification.html

not sure what it is changing exactly.
I guess it is best to try to use the latest version first and then if we encounter issues we might choose a specific one, what do you think?

Reply:
Most Java parsers are compatible with draft-03 and draft-04, that's why I've used that version. Better checking the eclipse tooling compatibility